docs/source/configuration.rst

```diff
@@ -335,6 +335,19 @@ This relationship can further be maintained in the JSON-LD structured data using
             ssn: "http://www.w3.org/ns/ssn/"
             Datastream: sosa:isMemberOf
 
+Sometimes, the JSON-LD desired is more complicated than can be achieved by simply aliasing properties using a
+context. In thise case, it is possible to specify a jinja2 template. When ``template`` is defined for a feature
+collection, the json-ld prepared by pygeoapi will be used to render a jinja2 template. pygeoapi provides a default
+template, which will render the unmodified json-ld document.
+
+.. code-block:: yaml
+
+    template: jsonld/base.jsonld
+    context:
+        - datetime: https://schema.org/DateTime
+
```

Comment:
Add a note that the path can the template path can be relative to the location of the configuration, or absolute.  Also add that one can use environment variables to have better deployment flexibility of template locations